doctrine.rst

```diff
@@ -638,10 +632,11 @@ Fetching an object back out of the database is even easier. For example,
 suppose you've configured a route to display a specific ``Product`` based
 on its ``id`` value::
 
-    public function showAction($productId)
+    use Doctrine\ORM\EnityManagerInterface;
+
+    public function showAction($productId, EnityManagerInterface $em)
     {
-        $product = $this->getDoctrine()
-            ->getRepository('AppBundle:Product')
+        $product = $em->getRepository('AppBundle:Product')
```

Comment:
Yes, we should start doing this :). But it's gotta be in a different PR, because it needs to be applied to 3.1 and higher